### PR TITLE
PG buffer profile based on port configuration

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**

### CODE REVIEW

**Comment by andriymoroz-mlnx** [comment]
related PR https://github.com/Azure/sonic-swss/pull/301

File: dockers/docker-orchagent/start.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ mkdir -p /etc/swss/config.d/
 sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/ipinip.json.j2 > /etc/swss/config.d/ipinip.json
 sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/mirror.json.j2 > /etc/swss/config.d/mirror.json
 sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/ports.json.j2 > /etc/swss/config.d/ports.json
+sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/msn27xx.32ports.buffers.json.j2 > /etc/swss/config.d/msn27xx.32ports.buffers.json
```
Comment (stcheng) [inline dockers/docker-orchagent/start.sh]: better to move this to the platform specific part below in `swssconfig.sh` file?

Comment (andriymoroz-mlnx) [reply dockers/docker-orchagent/start.sh]: agree

**Comment by andriymoroz-mlnx** [comment]
New PR will be created after https://github.com/Azure/sonic-swss/pull/319 is merged

**Pull request closed without merging**